syz-ci/manager.go

```diff
@@ -272,6 +272,9 @@ func (mgr *Manager) pollAndBuild(lastCommit string, latestInfo *BuildInfo) (
 				buildSem.Signal()
 			case <-mgr.stop:
 			}
+		} else {
+			// There was nothing to update.
+			success = true
```

Comment:
We will falsely set `success = true` on the second poll after unsuccessful build due to `commit.Hash != lastCommit` check.

Reply:
Thanks, indeed!

I've adjusted the code.